runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -723,6 +793,35 @@ private void removeCounter()
 		counter = null;
 	}
 
+	private void addBraceletCounter()
+	{
+		if (!config.showBraceletInfobox() || braceletCounter != null
+			|| (!equippedSlaughter && !equippedExpeditious))
+		{
+			return;
+		}
+
+		int braceletID = equippedSlaughter ? ItemID.BRACELET_OF_SLAUGHTER : ItemID.EXPEDITIOUS_BRACELET;
+		int braceletCount = equippedSlaughter ? slaughterChargeCount : expeditiousChargeCount;
+		String braceletTooltip = equippedSlaughter ? "Bracelet of slaughter" : "Expeditious bracelet";
+
+		braceletCounter = new Counter(itemManager.getImage(braceletID), this, String.valueOf(braceletCount));
+		braceletCounter.setPriority(InfoBoxPriority.MED);
```

Comment:
This do not needs priority? (idk since when we have this anyway)

Reply:
The priority is set so the task counter and bracelet infoboxes always show up in the same order. Without it they'll be ordered based on whichever one was added more recently

I can remove it if you prefer but I think it's good to keep them consistent